generated/nidaqmx/system/watchdog.py

```diff
@@ -374,6 +375,7 @@ def close(self):
         self._interpreter.clear_task(self._handle)
 
         self._handle = None
+        self._close_on_exit = False
```

Comment:
On further thought, this line changes behavior, so please delete it. This applies to both `Task` and `WatchdogTask`.

Explanation:

With the prior release, calling `close` twice prints a warning:
```
>>> import nidaqmx
>>> task = nidaqmx.Task()
>>> task.close()
>>> task.close()
D:\dev\measurementlink-python\examples\nidaqmx_analog_input\.venv\lib\site-packages\nidaqmx\task.py:456: ResourceWarning: Attempted to close NI-DAQmx task of name "_unnamedTask<0>" but task was already closed.
  warnings.warn(
```

It also prints a warning if the context manager calls `close` the 2nd time:
```
>>> with nidaqmx.Task() as task:
...     task.close()
...
D:\dev\measurementlink-python\examples\nidaqmx_analog_input\.venv\lib\site-packages\nidaqmx\task.py:456: ResourceWarning: Attempted to close NI-DAQmx task of name "_unnamedTask<1>" but task was already closed.
  warnings.warn(
```

If `close` clears `self._close_on_exit`, then the 2nd snippet will no longer print a warning. This allows users to double-close the task when using a context manager.

Hypothetical: If we wanted to allow multiple calls to `close`, the right way to do it would be to remove the warning.